pom.xml

```diff
@@ -21,5 +21,23 @@
 			<version>4.8.1</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
We must get this dependency to test a REST Endpoint in a standalone mode. If you are in the web context, you don't need that.